components/scoreBoard/scoreBoard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styles from './scoreBoard.module.scss'
+
+// interface ScoreBoardProps {
```

Comment:
Instead of props we could write this interface like we would do on BCI with just {score: number, level: number} and then pass those 2 props down to a  functional component that would extend this interface something like `const Scoreboard: React.FC<ScoreBoardProps> = ({score, level}) =>{'